Userland/Applications/Calendar/EventCalendar.cpp

```diff
@@ -33,17 +33,18 @@ void EventCalendar::paint_tile(GUI::Painter& painter, GUI::Calendar::Tile& tile,
         events.for_each([&](JsonValue const& value) {
             auto const& event = value.as_object();
 
-            if (!event.has("start_date"sv) || !event.has("summary"sv))
+            if (!event.has("start_date"sv) || !event.has("start_date"sv) || !event.has("summary"sv))
                 return;
 
             auto start_date = event.get("start_date"sv).value().to_deprecated_string();
+            auto start_time = event.get("start_time"sv).value().to_deprecated_string();
             auto summary = event.get("summary"sv).value().to_deprecated_string();
+            auto combined_text = DeprecatedString::formatted("{} {}", start_time, summary);
 
             if (start_date == DeprecatedString::formatted("{}-{:0>2d}-{:0>2d}", tile.year, tile.month, tile.day)) {
 
-                auto text_rect = tile.rect.translated(4, 4 + (font_height + 8) * ++index);
-
-                painter.draw_text(text_rect, summary, Gfx::FontDatabase::default_font(), Gfx::TextAlignment::TopLeft, palette().base_text());
+                auto text_rect = tile.rect.translated(4, 4 + (font_height + 4) * ++index);
+                painter.draw_text(text_rect, combined_text, Gfx::FontDatabase::default_font(), Gfx::TextAlignment::TopLeft, Gfx::Color::Red, Gfx::TextElision::Right);
```

Comment:
Using `Red` here instead of one of the Palette colours means it might not work well with all themes. Any reason for not using `base_text()` like before?

Reply:
Yes you are right, it will be better to keep `base_text()`. I didn't realize it can effect other themes.